common/src/main/scala/com/gu/zuora/model/SubscriptionData.scala

```diff
@@ -0,0 +1,31 @@
+package com.gu.zuora.model
+
+import org.joda.time.LocalDate
+
+case class SubscriptionData(ratePlanData: List[RatePlanData], subscription: Subscription)
+
+case class Subscription(
+  contractEffectiveDate: LocalDate,
+  contractAcceptanceDate: LocalDate,
+  termStartDate: LocalDate,
+  autoRenew: Boolean = true,
+  //scalastyle:off magic.number
+  initialTerm: Int = 12,
```

Comment:
Yes but this way you save one character! 

I think it might be good to switch the magic number inspection off, it's  a very blunt instrument currently